src/components/NewClusterModal.js

```diff
@@ -985,7 +1004,15 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
         // TODO PD: What do we do with this?
         !sparkMode && !isPersistentDisk && div([
           p(['Time to upgrade your compute runtime. Terra’s new persistent disk feature will safegard your work and data.']),
-          h(Link, { onClick: () => this.setState({ viewMode: 'aboutPersistentDisk' }) }, ['Learn more'])
+          h(Link, {
+            onClick: async () => {
+              this.setState({ viewMode: 'aboutPersistentDisk' })
+              await Ajax().Metrics.captureEvent(Events.aboutPersistentDiskView, {
```

Comment:
TOL: There's no harm in the `await` here, but we normally just fire-and-forget metrics calls. Is there a need for it here?

Reply:
ah nope I will remove that